src/Microsoft.Diagnostics.Monitoring.RestServer/Models/EventPipeProviderModel.cs

```diff
@@ -15,12 +16,12 @@ public class EventPipeProviderModel
         [DataMember(Name = "name", IsRequired = true)]
         public string Name { get; set; }
 
-        [DataMember(Name = "keywords", IsRequired = true)]
+        [DataMember(Name = "keywords")]
         [IntegerOrHexString]
-        public string Keywords { get; set; }
+        public string Keywords { get; set; } = "0x" + EventKeywords.All.ToString("X");
```

Comment:
I believe you mentioned these defaults match dotnet-trace, correct?

Reply:
Correct, dotnet-trace uses -1 (EventKeywords.All or 0xFFFFFFFFFFFFFFFF in terms of dotnet-monitor understands) as the default for keywords and uses Verbose as the default for event level.